### PR TITLE
version 2.0.4 for PhpStorm 2024.1

### PR DESCRIPTION
**Pull request by Hidanio**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -25,7 +28,7 @@ jobs:
       - name: Gradle Wrapper Validation
         uses: gradle/wrapper-validation-action@v1.1.0
 
-      # Set up Java environment for the next steps
+          # Set up Java environment for the next steps
```
Comment (Danil42Russia) [inline .github/workflows/build.yml:31]: This is definitely an unnecessary action

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,12 @@
 
 ## [Unreleased]
 
+## 2.0.4 - 09.04.2024
+
+- Adapt code for PhpStorm 2024.1
+- More readable warnings & error while connecting to servers
```
Comment (Danil42Russia) [inline CHANGELOG.md:8]: Let's not do that in this release

File: .github/workflows/build.yml (hunk 1 above)
Comment (Hidanio) [reply .github/workflows/build.yml]: Fixed

File: CHANGELOG.md (hunk 2 above)
Comment (Hidanio) [reply CHANGELOG.md]: Removed

**Review by Danil42Russia — APPROVED** [review]

**Pull request merged**